easybuild/easyconfigs/s/SuiteSparse/AMD_GNUMakefile-4.4.5.patch

```diff
@@ -0,0 +1,42 @@
+--- SuiteSparse/AMD/Lib/GNUmakefile.org	2014-03-21 20:22:00.000000000 +0100
```

Comment:
please include a comment at the top describing what this patch does, and who the author is